tests/torchtune/models/clip/test_positional_embeddings.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import pytest
+
+import torch
+
+from tests.test_utils import assert_expected
+from torchtune.models.clip._position_embeddings import (
+    TiledTokenPositionalEmbedding,
+    TilePositionalEmbedding,
+    TokenPositionalEmbedding,
+)
+
+from torchtune.utils.seed import set_seed
+
+
+class TestVisionTransformer:
+    @pytest.fixture(autouse=True)
+    def setup_class(self):
+
+        self.embed_dim = 16
+        self.tile_size = 14
+        self.max_num_tiles = 3
+        self.batch_size = 2
+        self.patch_size = 2
+        self.aspect_ratio = torch.tensor([[3, 1], [1, 2]])
+        self.patch_grid_size = self.tile_size // self.patch_size
+
+        set_seed(42)
+        self.input_tensor = torch.randn(
+            (
+                self.batch_size,
+                self.max_num_tiles,
+                self.patch_grid_size**2 + 1,
+                self.embed_dim,
+            )
+        )
+
+    def test_token_positional_embedding(self):
+        # call model
+        set_seed(42)
```

Comment:
More of a nit, but we try to use non-random initialization as much as possible in our tests. We have [this utility](https://github.com/pytorch/torchtune/blob/f1585774eb52ee25837edc6c6f4856c6ed50503b/tests/test_utils.py#L137) to do this. Main reason being that if for any reason we change the order submodules are initialized it'll break this test (such changes happen more often than you would think). 

Separately, if you are setting seed like this you can do it in its own fixture rather than in every unit test (I think your other unit test file does this already).